service/service.go

```diff
@@ -68,9 +66,8 @@ func (svc *Service) HandleMessage(ctx context.Context, message kafka.Message) (s
 		if err := recover(); err != nil {
 			log.Event(ctx, "panic in handle message", log.ERROR,
 				log.Data{
-					"err":      fmt.Sprintf("%+v", err),
-					"err_spew": spew.Sdump(err),
-					"stack":    string(debug.Stack()),
+					"err":   fmt.Sprintf("%+v", err),
+					"stack": string(debug.Stack()),
```

Comment:
Do we still want to leave the app die here? Once we've output the error details I think it would be better to allow the app to die/exit. 
It dying will trigger platform alerts and health check warning alerting us to the issues. I think swallowing the error is masking the fact something something has gone pretty badly wrong.

Reply:
My thinking was that a panic inside the handler would most likely be a message specific issue, instead of an app issue. In this particular case it was a message specific issue, so the endless restart loop was preventing other messages from being processed. I came to this conclusion also based on how the import process is used - we generally fail fast on the import, and leave it in an unfinished state.

I can see your point though, it could still be something not message specific thats failing, which would only be recognised by someone investigating the failed import. Do you think we should stick with the current behaviour of exiting the app?

Reply:
I have removed the panic handling as suggested. It's better to ensure that other panics are not masked, as we cannot be sure that a panic is message specific. Turns out the panic is logged but it goes to stderr, and requires an additional flag in the nomad logs command to show it 🤦 `nomad alloc logs -stderr`